README.md

```diff
@@ -18,6 +18,8 @@ Material Components for the web is the successor to [Material Design Lite](https
   - [Blox Material](https://blox.src.zone/material): Angular Integration Library.
   - [Vue MDC Adapter](https://github.com/stasson/vue-mdc-adapter): MDC Web Integration for Vue.js (using [foundation/adapters](./docs/integrating-into-frameworks.md#the-advanced-approach-using-foundations-and-adapters).)
   - [Material Components Vue](https://github.com/matsp/material-components-vue): MDC Web Integration for Vue.js (using [vanilla components](./docs/integrating-into-frameworks.md#the-simple-approach-wrapping-mdc-web-vanilla-components))
+  - [Material Components React](https://github.com/material-components/material-components-web-react): MDC Web Integration for React.js (using [foundation/adapters](./docs/integrating-into-frameworks.md#the-advanced-approach-using-foundations-and-adapters))
```

Comment:
Should we consider putting these at the top given that they're first-party whereas the others are third-party? Maybe even have a separate sub-bullet for third-party integrations?

e.g.

- Seamless integration with other JS frameworks and libraries
  - Material Components for React (note the presence of "for", too)
  - Material Web Components
  - Additional third-party integrations
    - (rest of list here)